File: Programar em C#/17-Ciclo-foreach.cs
// Programar em Python #17 - Ciclo foreach

using System;
using System.Collections.Generic;

namespace Base {
    class CicloForeach {
        public void Run() {
            string organizacao = "Caffeine Algorithm";
            List<string> paises = new List<string>() {
                "Portugal",
                "Brasil",
                "Espanha",
                "França",
                "Itália",
                "Austrália",
                "Índia"
            };

            foreach (char carater in organizacao)
                Console.WriteLine($"Carater: {carater}");

            foreach (string pais in paises)
                Console.WriteLine($"País: {pais}");
        }
    }
}

File: Programar em Java/17-Ciclo-foreach.java
// Programar em Java #17 - Ciclo "foreach"

package com.caffeinealgorithm.programaremjava;

import java.util.List;
import java.util.ArrayList;

public class CicloForeach {
    public void Run() {
        List<String> paises = new ArrayList<>();

        paises.add("Portugal");
        paises.add("Brasil");
        paises.add("Espanha");
        paises.add("França");
        paises.add("Itália");
        paises.add("Austrália");
        paises.add("Índia");

        for (String pais : paises)
            System.out.println(new StringBuilder().append("País: ").append(pais));
    }
}